Gradle build task failure for :x-pack:qa:saml-idp-tests:setupPorts

All the failures related to the corretto JVM except for one:
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+corretto-periodic/ES_BUILD_JAVA=openjdk12,label=amazon/91/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.0+corretto-periodic/ES_BUILD_JAVA=openjdk12,label=amazon/91/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+corretto-periodic/ES_BUILD_JAVA=openjdk12,label=amazon/190/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.7+corretto-periodic/ES_BUILD_JAVA=openjdk12,label=amazon/94/console

And one not corretto:
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.0+multijob-darwin-compatibility/88/console

All failed with:

Full failure
```
> Task :x-pack:qa:saml-idp-tests:setupPorts FAILED
Task ':x-pack:qa:saml-idp-tests:setupPorts' is not up-to-date because:
  Task has not declared any outputs despite executing actions.
Custom actions are attached to task ':x-pack:qa:saml-idp-tests:setupPorts'.
:x-pack:qa:saml-idp-tests:setupPorts (Thread[Daemon worker,5,main]) completed. Took 0.003 secs.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':x-pack:qa:saml-idp-tests:setupPorts'.
4649 actionable tasks: 2449 executed, 2200 up-to-date

> Cannot get property 'test.fixtures.shibboleth-idp.tcp.4443' on extra properties extension as it does not exist
```

This seems to have been introduced in PR: https://github.com/elastic/elasticsearch/pull/40333

--- a/x-pack/qa/saml-idp-tests/build.gradle
+++ b/x-pack/qa/saml-idp-tests/build.gradle
@@ -38,6 +38,9 @@ task setupPorts {
         idpMetaFile.write(content.toString(), "UTF-8")
     }
 }
+// Don't attempt to run integTest when Docker is not available
+integTest.enabled = idpFixtureProject.postProcessFixture.enabled
+
 integTestCluster.dependsOn setupPorts
 
 integTestCluster {

--- a/x-pack/qa/saml-idp-tests/build.gradle
+++ b/x-pack/qa/saml-idp-tests/build.gradle
@@ -38,8 +38,8 @@ task setupPorts {
         idpMetaFile.write(content.toString(), "UTF-8")
     }
 }
-// Don't attempt to run integTest when Docker is not available
-integTest.enabled = idpFixtureProject.postProcessFixture.enabled
+// Don't attempt to get ephemeral ports when Docker is not available
+setupPorts.onlyIf { idpFixtureProject.postProcessFixture.enabled }
 
 integTestCluster.dependsOn setupPorts
 

